tests/api_v2/test_cache.py

```diff
@@ -12,10 +12,6 @@
 import sys
 
 from tank_test.tank_test_base import setUpModule
-
-# import the test base class
-test_python_path = os.path.abspath(os.path.join( os.path.dirname(__file__), "..", "python"))
```

Comment:
`run_tests.py` adds the `python` folder of other repos now, no need to do that. :)